invenio_users_resources/profiles/schemas.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 TU Wien.
+#
+# Invenio-Users-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Schemas for user preferences with fields required by this module."""
+
+from invenio_accounts.profiles import \
+    UserPreferencesSchema as BasePreferencesSchema
+from marshmallow import fields
+
+from ..services.schemas import validate_visibility
+
+
+class UserPreferencesSchema(BasePreferencesSchema):
```

Comment:
Comment: Should this be in Invenio-Accounts?

Reply:
sure, we can put this in invenio-accounts and save us some hassle w.r.t. setting a default value in `invenio.cfg`
the only reason why i put this here was that we don't need it anywhere in invenio-accounts